tests/Microsoft.VisualStudio.ProjectSystem.Managed.TestServices/AssertEx.cs

```diff
@@ -8,14 +8,7 @@ namespace Xunit;
 internal static class AssertEx
 {
     public static void CollectionLength<T>(IEnumerable<T> collection, int expectedCount)
-    {
-        int actualCount = collection.Count();
-
-        if (actualCount != expectedCount)
-        {
-            throw new CollectionException(collection, expectedCount, actualCount);
-        }
-    }
+        => Assert.Equal(expectedCount, collection.Count());
```

Comment:
This will give worse error messages when failures occur. Let's use the following instead.

```
throw CollectionException.ForMismatchedItemCount(expectedCount, actualCount, "Collection lengths not equal.");
```